packages/nextjs/README.md

```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
Not sure if we really need this file @Nadai2010 @metalboyrick . Can we remove even from scaffold-stark?

Reply:
ya can remove